pytest_astropy_header/display.py

```diff
@@ -120,9 +120,8 @@ def pytest_report_header(config):
 
     s += "\n"
 
-    special_opts = ["remote_data", "pep8"]
     opts = []
-    for op in special_opts:
+    for op in ("remote_data", "pep8", "run_slow", "run_hugemem"):
```

Comment:
Hmm, not sure what pep8 is for either. Maybe added back when flake8 was not a thing?